cmd/dp-import-tracker/main.go

```diff
@@ -659,7 +659,7 @@ func main() {
 		if err = newInstanceEventConsumer.Close(shutdownContext); err != nil {
 			log.ErrorC("bad close", err, log.Data{"topic": cfg.NewInstanceTopic})
 		}
-		if err = store.Close(shutdownContext); err != nil {
+		if err = graphDB.Close(shutdownContext); err != nil {
```

Comment:
Does this always have to be closed? If so would it need to be a `defer`?

Reply:
it _should_ always close, but as it's done on program shutdown it'll get killed anyway if it has an issue. I'm not sure a whole lot would be gained from defering it, and we'd have to defer a func that includes the error handling.

I agree that usually when we're closing things we defer to make sure it happens, but I dont know whether its different here because everything it's doing is closing connections to resource, or just that it needs to be updated 🤔